### DISCUSSION TITLE
Youtube search age-restricted videos bypass

### DISCUSSION THREAD

**Discussion by ShepoPL** [zerodytrash/Simple-YouTube-Age-Restriction-Bypass#150] [Ideas]
Can you make a bypass for blocking age-restricted videos from showing in youtube search results? I would be glad if you make it.

**Comment by ghost** [comment]
I asked that question after an issue I put up was resolved and drunkwinter said that he doesn't think it's possible (I'm not a contributor of the project). I know a youtube website that makes the age-restricted videos available to watch (most of the time) and searching for it works there youtube.com will have to be replaced with yewtu.be, after clicking the videos, press "Watch on YouTube" on the left because the settings of the videos are weird there and there could be a few times the website won't work for a while... https://yewtu.be/feed/popular

**Reply by drunkwinter** [reply]
> I know a youtube website that makes the age-restricted videos available to watch (most of the time) and searching for it works there

Yes, they most likely do their own search indexing. This is something we realistically cannot do, since we don't store any videos or any search queries of what people type on our server. If we were to do that, it would cost x amount of money, which would have to come from somewhere.
